Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Frostmourne

## Changelog:
- Added this thing.
- Changed that thing.


## Developer changelog:
- Added this thing.
- Changed that thing.

# Tasks:
- [x] Add icon
- [x] Add name and desc
- [x] Add modifiers
- [x] Make sure you can't gift Frostmourne
- [x] Make sure Frostmourne can't lose durability
- [x] The Frostmourne makes you not able to refuse the Lich King
- [x] Link Frostmourne via `artifact = { }` in events
- [x] Make Frostmourne equipped during the events
- [x] Spawn Frostmourne if Arthas couldn't find it
- [ ] Event: someone is killed by Frostmourne, you may resurrect him with a twisted soul (don't mention sword plunged or combat, it's a part of the combat events)
  - Option 1: turn undead, remove positive traits, add negative traits like Callous, Vengeful
  - Option 2: let it be dead
  - [ ] Write text, 600-500 characters, in 1st person, from the killer perspective
- [ ] Change text of combat events to represent resurrection


## Tests:


# How to test:
Make sure Frostmourne can't be gifted and destroyed. Test Arthas' events to make sure Arthas gets Frostmourne and has it in the event windows.
